CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 3.1.1 (2019-09-11)
+
+- Fixed the summary action bar component to properly style the body when the action bar expands and collapses. [#18](https://github.com/blackbaud/skyux-action-bars/pull/18)
```

Comment:
Is there a way to be more specfic? Instead of "to -properly style", how about something like "to have proper bottom margin"?